src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <header class="container">
+      <a href="#" class="img__logo">
+        <img src="/src/images/logo.png"
+        class="logo" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link link1 is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link link2">Samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link link1 is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link link2">Samsung</a>
          </li>
```

Add empty line between multiple sibling blocks of html